source/less/blocks/hero.less

```
@@ -0,0 +1,36 @@
+.hero {
+  background-image: url("../img/hero-mobile@2x\ 1.jpg");
```

```suggestion
  background-image: url("../img/hero-mobile@2x.jpg");
```
Про переименование этого файла писал выше. Названия файлов не должны содержать пробелов и заглавных букв.